devtools-html/debugger.html
# Show original files when the files have the same URL #5663

Fixes Issue: #5663 Show original files when the files have the same URL

Here's the Pull Request Doc
https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests

### Summary of Changes

* modify `SourcesTree.js` and `addToTree.js` to support adding the same filename and path
* upgrade lint-staged to 7.0.5 which supports node 10

### Test Plan

1. Open https://q7mmol9xy9.codesandbox.io/
2. Open devtools.
3. Expand src/ folder.
4. Verify: `Hello.js` appears twice.
5. Verify: `index.js` appears twice.
6. Verify: Click `Hello.js` and verify generated js file is shown.
7. Verify: Click second `Hello.js` and verify original source js file is shown.

CURRENT BUG: clicking requires two clicks... might be key mismatch on React tree component?

Here's the Debugger's Testing doc
https://docs.google.com/document/d/1oBMRxV8A2ag2t22YsQOxTdEv0mXKzIg0tggJjRkU1S0/edit#.
Feel free to improve it!

### Screenshots/Videos (OPTIONAL)
